src/website/views/components/data-table/_examples.pug

```diff
@@ -0,0 +1,2712 @@
+            .chi-pagination__center
+              .chi-pagination__button-group.chi-button-group
+                a(class=`chi-button -icon -flat`, href='#', aria-label='Previous page')
+                  .chi-button__content
+                    i(class="chi-icon icon-chevron-left", aria-hidden='true')
+                a(class=`chi-button -flat`, href='#') 1
+                div(class=`chi-button -flat -disabled`) ...
+                a(class=`chi-button -flat`, href='#') 3
+                a(class=`chi-button -flat`, href='#') 4
+                a(class=`chi-button -flat -active`, href='#') 5
+                a(class=`chi-button -flat`, href='#') 6
+                a(class=`chi-button -flat`, href='#') 7
+                div(class=`chi-button -flat -disabled`) ...
+                a(class=`chi-button -flat`, href='#') 12
+                a(class=`chi-button -icon -flat`, href='#', aria-label='Next page')
+                  .chi-button__content
+                    i(class="chi-icon icon-chevron-right", aria-hidden='true')
+            .chi-pagination__end
+              .chi-pagination__jumper
+                label.chi-pagination__label(for="jumper-input-1") Go to page:
+                input(type='text', class='chi-input', id='jumper-input-1')
+              
+  .example-tabs.-pl--2
+    ul.chi-tabs#example-menu-base
+      li.-disabled
+        a(href=`#webcomponent-menu-base`) Web Component
+      li.-active
+        a(href=`#html-menu-base`) HTML Blueprint
+  .chi-tabs-panel#webcomponent-menu-base
+  .chi-tabs-panel.-active#html-menu-base
+    :code(lang="html")
+      <div class="chi-data-table">
+      </div>
+
+h3 Base -- Compact
+.example.-mb--3
+  .-p--3.-p-md--5
+    .chi-data-table.-compact
+      .chi-data-table__head
+        .chi-data-table__row
+          .chi-data-table__cell
+            span Name
+          .chi-data-table__cell
+            span ID
+          .chi-data-table__cell
+            span Last Login
+      .chi-data-table__body
+        .chi-data-table__row
+          .chi-data-table__cell.-name.-bold
+            div Name 1
+          .chi-data-table__cell.-name
+            div name-1
+          .chi-data-table__cell
+            div Dec 18, 2020 3:26 PM
+        .chi-data-table__row
+          .chi-data-table__cell.-name.-bold
+            div Name 2
+          .chi-data-table__cell.-name
+            div name-2
+          .chi-data-table__cell
+            div Dec 18, 2020 2:38 AM
+        .chi-data-table__row
+          .chi-data-table__cell.-name.-bold
+            div Name 3
+          .chi-data-table__cell.-name
+            div name-3
+          .chi-data-table__cell
+            div Nov 5, 2020 10:15 AM
+      .chi-data-table__footer
+        nav.chi-pagination.-compact(role='navigation', aria-label='Pagination')
+          .chi-pagination__content
+            .chi-pagination__start
+              .chi-pagination__results
+                span.chi-pagination__label 240 Results
```

Comment:
@jllr This new placement will require a minor change to pagination styles to eliminate the margin-left on the results label. The current compact results label renders a margin-left as it assumes there will be content on the left side. With the new placement, the results label will always be the first child so no margin-left is necessary.
![image](https://user-images.githubusercontent.com/130654/97204260-1e2ee600-1773-11eb-9d84-8199d66c0389.png)